Resources/doc/reference/architecture.rst

```diff
@@ -354,6 +354,32 @@ Then, you have to set the CommentAdmin ``parentAssociationMapping`` attribute to
         }
     }
 
+To display the CommentAdmin extend the menu in your ``PostAdmin`` class:
+
+.. code-block:: php
+
+    <?php
+    namespace Sonata\NewsBundle\Admin;
+
+    class PostAdmin extends Admin
+    {
+        // ...
+
+        protected function configureSideMenu(MenuItemInterface $menu, $action, AdminInterface $childAdmin = null)
+        {
+            if (!$childAdmin && !in_array($action, array('edit', 'show'))) { return; }
+
+            $admin = $this->isChild() ? $this->getParent() : $this;
+            $id = $admin->getRequest()->get('id');
+
+            $menu->addChild('Show Post', array('uri' => $admin->generateUrl('show', array('id' => $id))));
```

Comment:
Personally, I don't use the admin, but the security context (I did not manage to change that to the authorization checker, due to [this problem](http://stackoverflow.com/questions/32590621/tokenstorage-sometimes-returns-null-in-service)). Not sure using the admin is the best way to go, because the menu can link to many admins, which would mean you would have to get all these admins to call `isGranted` on them.